src/Microsoft.Health.Dicom.Api/Controllers/CustomTagController.cs

```diff
@@ -50,5 +51,51 @@ public async Task<IActionResult> PostAsync([FromBody] IEnumerable<CustomTagEntry
             return StatusCode(
                (int)HttpStatusCode.Accepted, response);
         }
+
+        /// <summary>
+        /// Handles requests to get all custom tags.
+        /// </summary>
+        /// <returns>
+        /// Returns Bad Request if given path can't be parsed. Returns Not Found if given path doesn't map to a stored
+        /// custom tag or if no custom tags are stored. Returns OK with a JSON body of all tags in other cases.
+        /// </returns>
+        [ProducesResponseType(typeof(JsonResult), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [HttpGet]
+        [Route(KnownRoutes.CustomTagRoute)]
+        [AuditEventType(AuditEventSubType.GetAllCustomTags)]
+        public async Task<IActionResult> GetAllTagsAsync()
+        {
```

Comment:
Based on https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#93-collection-url-patterns, looks like we should combine getCustomTag and get all custom tag them together?

Reply:
resolved offline!